HWIMO-BUILD

```diff
@@ -23,6 +23,14 @@ pushd packagebuild
 mkdir common
 mkdir pxe
 
+#add Microkernel images from bintray
+pushd $output_path/builds/
+wget $bintray_repo/base.trusty.3.16.0-25-generic.squashfs.img
```

Comment:
why using wget instead of "rebuild" ? are you relying on TravisCI to build them and upload them ?

In case that there's code change , there will be a race-condition that before TravisCI done and CI runs into this step.

Reply:
@panpan0000 I am planing to deprecate the mikrokernel build.  There is a number of workflows that needs to be changed first.

Reply:
make sense if these part of code will be retired soon.
glad to hear that only 1 kind of micro-kernel will remain.